compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -483,7 +483,10 @@ loco::NodeShape infer_batchmatmul_shape(const loco::TensorShape &x_shape,
   loco::Dimension y_rhs = adj_y ? y_shape.dim(y_rank - 2) : y_shape.dim(y_rank - 1);
 
   if (x_rhs.known() && y_lhs.known() && not(x_rhs == y_lhs))
```

Comment:
Is it okay for `x_rhs != y_lhs` ? TFLite interpreter (and circle interpreter) does not allows such a shape.

Reply:
I think the cause is from circle-mlir which seems fix is needed.
Thanks for taking a look at tflite interpreter codes :)